python/JSONtoObj.c

```diff
@@ -144,7 +144,27 @@ static void Object_releaseObject(void *prv, JSOBJ obj)
 
 //static char *g_kwlist[] = {"obj", NULL};
 
-HPy_DEF_METH_O(JSONToObj)
+HPyDef_METH(JSONToObj, "loads", JSONToObj_impl, HPyFunc_O,
+            .doc="Converts JSON as string to dict object structure. "
+                 "Use precise_float=True to use high precision float decoder.");
+
+// ujson.c does something a bit weird: it defines two Python-level methods
+// ("encode" and "loads") for the same C-level function
+// ("JSONToObj_impl"). HPyDef_METH does not support this use case, but we can
+// define our HPyDef by hand: HPyDef_METH is just a convenience macro and the
+// structure and fields of HPyDef is part of the publich API
+HPyDef JSONToObj_decode = {
```

Comment:
We previously had `HPyMeth_Decl` (or something like this). I think the problem when using `HPyDev_METH` multiple times is that we it would re-define the signature type, right? We could make `HPyDef_METH` repeatable.